app/javascript/gobierto_plans/webapp/components/GroupsByTermTableRow.vue

```diff
@@ -10,7 +10,7 @@
       >
         <i :class="['fas', nestedGroups.length ? `fa-caret-${isOpen ? 'down' : 'right'}` : '']" />
         <router-link
-          :to="{ name: 'term', params: { ...params, term: slug } }"
+          :to="{ name: routes.TERM, params: { ...params, term: slug } }"
```

Comment:
I like this way of naming routes

Reply:
Two best practices here:
1. Avoid text literals in the code
2. At the moment that one variable is reused somewhere else..... move it to shared place